remoulade/middleware/current_message.py

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: LPGL-3.0
+# Copyright (C) 2019 CLEARTYPE SRL <bogdan@cleartype.io>
+# Based in current_message.py from the dramatiq project
+
+from threading import local
+
+from .middleware import Middleware
+
+
+class CurrentMessage(Middleware):
```

Comment:
Add CurrentMessage to default_middleware (in middleware __init__.py), It's the list of middleware which are automatically loaded with remoulade. As we have no configuration for this middleware we can enable it for everyone.